dali/operators/generic/constant.h

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_OPERATORS_GENERIC_CONSTANT_H_
+#define DALI_OPERATORS_GENERIC_CONSTANT_H_
+
+#include <vector>
+
+#include "dali/pipeline/operator/operator.h"
+#include "dali/core/tensor_view.h"
+#include "dali/core/static_switch.h"
+
+#define CONSTANT_OP_SUPPORTED_TYPES \
+  (bool, int8_t, uint8_t, int16_t, uint16_t, int32_t, uint32_t, int64_t, uint64_t, float)
```

Comment:
wondering, how about float16? It might be useful if we want to make a quick synthetic pipeline producing constant data just to measure the training speed without preprocessing

Reply:
Would need special treatment for GPU code to compile, but yeah, I can try.